fiware_cloto/cloto_settings/settings.py

```diff
@@ -151,6 +151,7 @@
 AUTH_API = config.get('openstack', 'AUTH_API')
 
 # POLICY MANAGER CONFIGURATION
+SECURITY_LEVEL = config.get('policy_manager', 'SECURITY_LEVEL')
 SETTINGS_TYPE = config.get('policy_manager', 'SETTINGS_TYPE')
 INSTALLATION_PATH = os.path.dirname(fiware_cloto.__file__)
 ENVIRONMENTS_MANAGER_PATH = INSTALLATION_PATH + u'/environments/environmentManager.py'
```

Comment:
Are unicode strings needed in settings? I see not all properties are encoded this way.